plow_linter/src/lints/has_registry_license.rs

```diff
@@ -0,0 +1,83 @@
+use crate::lint::{
+    common_error_literals::{NO_ROOT_PREFIX, RDF_GRAPH_PARSE_ERROR},
+    helpers::{catch_single_annotations_which_must_exist, fail_if_contains_inappropriate_word},
+    lint_failure, lint_success, Lint, LintResult,
+};
+use harriet::TurtleDocument;
+use plow_graphify::document_to_graph;
+use plow_ontology::constants::REGISTRY_LICENSE;
+use plow_package_management::metadata::get_root_prefix;
+use rdftk_iri::IRI as RDFTK_IRI;
+use std::collections::HashSet;
+use std::str::FromStr;
+
+const RELATED_FIELD: &str = "`registry:license`";
+/// A sane character count for a `registry:license` field.
+const LICENSE_MAX_ALLOWED_CHAR_COUNT: usize = 100;
```

Comment:
100 chars for a license seems really low. It should be something that can accommodate a a license text ~the length of the full Apache 2.0 text (so maybe 10k characters?)

Reply:
Oh I've completely misunderstood the purpose of the field!
Now I understand that it is the content of the license. Got it. 
Probably 10k is fine 🤝

Reply:
[69aa86a](https://github.com/field33/plow/pull/4/commits/69aa86aca2b149dce9db72a3ef449ab7ea32fa7f)